modules/nlu/src/views/full/entities/ListEntity.tsx

```diff
@@ -71,7 +71,9 @@ export const ListEntityEditor: React.FC<Props> = props => {
   const isUnique = newElement =>
     !props.entities
       .filter(entity => entity.type === 'list')
-      .some(({ occurrences }) => occurrences.some(({ name, synonyms }) => [name, ...synonyms].includes(newElement)))
+      .some(({ occurrences }) =>
+        [...state.occurrences, ...occurrences].some(({ name, synonyms }) => [name, ...synonyms].includes(newElement))
```

Comment:
This works for sure but as we discussed, it would be better if you find the index of occurrences in state.occurrences and replace it for comparaison

Reply:
solved with _.uniq